pkgs/development/libraries/igraph/default.nix

```
@@ -86,7 +86,7 @@ stdenv.mkDerivation rec {
 
   preCheck = ''
     # needed to find libigraph.so
-    export LD_LIBRARY_PATH="$PWD/src"
+    export LD_LIBRARY_PATH="${lib.getLib blas}/lib:$PWD/src"
```

```suggestion
    export LD_LIBRARY_PATH="${lib.makeLibraryPath [ blas ]}:$PWD/src"
```
just in case we add something else in the future?

---

trying this now

---

This didn't work, but this did:

```
diff --git a/pkgs/development/libraries/igraph/default.nix b/pkgs/development/libraries/igraph/default.nix
index 911c77109c6..a060440456e 100644
--- a/pkgs/development/libraries/igraph/default.nix
+++ b/pkgs/development/libraries/igraph/default.nix
@@ -86,7 +86,8 @@ stdenv.mkDerivation rec {
 
   preCheck = ''
     # needed to find libigraph.so
-    export LD_LIBRARY_PATH="$PWD/src"
+    # export LD_LIBRARY_PATH="$PWD/src"
+    export DYLD_LIBRARY_PATH="${lib.makeLibraryPath [ blas ]}:$PWD/src"
   '';
 
   postInstall = ''
```

---

Integrated.